data_science_python.ipynb

```diff
@@ -579,13 +582,13 @@
      "collapsed": false,
      "input": [
       "def split_into_tokens(message):\n",
-      "    message = unicode(message, 'utf8')  # convert bytes into proper unicode\n",
+      "    message = str(message.encode('utf8'))  # convert bytes into proper unicode\n",
```

Comment:
-1: will break Python 2. Either detect the version from `sys.version`, or use a library like `six` to abstract.